docker/docker-compose.yml

```diff
@@ -38,4 +38,4 @@ services:
 networks:
   # This is the network created by gateway to enable communicaton between multiple docker-compose projects
   gateway_hubmap:
-    external: true
```

Comment:
@kburke we have to use the external docker network especially when we deploy on DEV/TEST/STGAE/PROD. Please revert this change.

Reply:
Setting reverted to true in branch file.